defaults-o2-daq.sh

```diff
@@ -32,6 +32,11 @@ overrides:
     tag: "v3.0.2"
   O2:
     version: "%(short_hash)s%(defaults_upper)s"
+  CMake:
+    version: "%(tag_basename)s"
+    tag: "v3.7.2"
+    prefer_system_check: |
+      which cmake && case `cmake --version | sed -e 's/.* //' | cut -d. -f1,2,3 | head -n1` in [0-2]*|3.[0-5].*) exit 1 ;; esac
```

Comment:
considered that 3.7 is used as tag, the system check should probably be `3.[0-6].*`